app/lib/cds_importer/entity_mapper/base_mapper.rb

```diff
@@ -54,7 +54,7 @@ def base_mapping
 
         # Register a callback to soft delete missing entities indicated by the passed in secondary mappers
         def delete_missing_entities(*secondary_mappers)
-          before_oplog_inserts do |xml_node, mapper_instance, primary_model_instance|
+          before_oplog_inserts do |xml_node, mapper_instance, primary_model_instance, _expanded_attributes|
```

Comment:
```suggestion
          before_oplog_inserts do |xml_node, mapper_instance, primary_model_instance|
```

The arguments we pass are dropped if the block arity is less than all of the arguments. No need to add this here